src/devices/Pcd8544/README.md

```diff
@@ -0,0 +1,156 @@
+# PCD8544 - 48 × 84 pixels matrix LCD
```

Comment:
I'd suggest put Nokia 5110 screen somewhere in the title to make it easier to find